### PR TITLE
Test email configuration functionality.

### PR DESCRIPTION
**Pull request by minnerthecat**
Added 'send test email' to logged in user drop down.

### CODE REVIEW

**Comment by nicmana** [comment]
I like it.

File: partEZ/app/Http/Controllers/Email/EmailController.php [hunk 1]
```diff
@@ -17,10 +18,24 @@ public function sendInvitation()
...
+
+    public function sendTestEmail() 
+    {
+
+        $data = array(
+            'firstname' => Auth::user()->firstname,
+        );
 
-            $message->to('nickmanaigre@gmail.com')->subject('Event Invitation');
+        Mail::send('emails.testemail', $data, function ($message) {
```
Comment (delroyhiebert) [inline partEZ/app/Http/Controllers/Email/EmailController.php:36]: @maxewaschuk just being picky but it looks like a tab is missing here...

**Comment by delroyhiebert** [comment]
@maxewaschuk wicked, looks beautiful!

File: partEZ/app/Http/Controllers/Email/EmailController.php (hunk 1 above)
Comment (minnerthecat) [inline partEZ/app/Http/Controllers/Email/EmailController.php:36]: @delroyhiebert nope! That's just the diff display glitching out, I promise everything is properly indented! :stuck_out_tongue:

**Pull request merged**